submissions/Lu-sure/Popup/index.html

```diff
@@ -0,0 +1,168 @@
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/apps.png" alt="Apps icon" class="popup-item-img">
+                      Apps
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/documents.png" alt="Documents icon" class="popup-item-img">
+                      Documents
+                    </a>
+                  </li>
+                </ul>
+                
+                <input type="checkbox" id="more-button">
+                <label for="more-button" class="more-bt-label"><span class="more-bt-text">More</span></label>
+
+                <hr class="popup-divider">
+
+                <ul class="popup-list add-popup">
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/chat.png" alt="Chat icon" class="popup-item-img">
+                      Chat
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/classroom.png" alt="Classroom icon" class="popup-item-img">
+                      Classroom
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/collection.png" alt="Collection icon" class="popup-item-img">
+                      Collection
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/mars.png" alt="Mars icon" class="popup-item-img">
+                      Mars
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/blogs.png" alt="Blogs icon" class="popup-item-img">
+                      Blogs
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/ads.png" alt="Ads icon" class="popup-item-img">
+                      Ads
+                    </a>
+                  </li>
+
+                  <li>
+                    <a class="popup-bottom" href="#">Even more from Kottans</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </li>
+
+          <li class="nav-img-item bell-item">
+            <a href="#">
+              <img class="bell-img" src="navigation-icons/bell.png" alt="">
+            </a>
+          </li>
+          <li class="nav-img-item">
+            <a href="#">
+              <img class="menu-img round-img" src="navigation-icons/user.jpg" alt="">
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+</body>
+</html>
```

Comment:
Add an empty line.

Reply:
No problem! I am just interested why is it needed?

Reply:
This is a feature of GitHub))

Reply:
There is a historical reason for this.
If you want, you can read here something about it here in the answers [Because that’s how the POSIX standard defines a line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725)

Reply:
Thanks.